descriptors/ModuleDescriptor-template.json

```diff
@@ -776,6 +776,17 @@
         }
       ]
     },
+    {
+      "id": "finance.finance-data",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/finance/finance-data",
+          "permissionsRequired": ["finance.finance-data.collection.get"]
```

Comment:
module permissions for acq units retrival should be added

Reply:
added permissions
```
 "acquisitions-units-storage.units.collection.
 "acquisitions-units-storage.memberships.collection.get"
```